src/cfml/system/modules_app/server-commands/commands/server/prune.cfc

```diff
@@ -0,0 +1,159 @@
+/**
+ * Prune.
+ * .
+ * {code:bash}
+ * server prune months=6
+ * server prune months=6 --list
+ * server prune months=6 --force
+ * {code}
+ **/
+component {
+
+	// di
+	property name="serverService" inject="ServerService";
+
+	/**
+	 * Prune.
+	 *
+	 * @months.hint forget servers which last started date is greater or equal to months you set
```

Comment:
Should we do days for more granularity in case someone wanted to do, say 2 weeks?  Also, will a decimal period of time work.  i.e. .75 months?

Reply:
i liked the month approach for simplicity, in days you have to start to do calculations in your head, but that is just me, years was to much, days is too little, months was just right i guessed.

Reply:
Months still feels a little to big for me.  As far as doing calculations in your head, it's pretty easy to just do 30, 60, 90 days for 1, 2, or 3 months.

Reply:
ok got it i will change it to days then

Reply:
now it uses days